guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,221 @@
+[id="Upgrading-Project-from-EL7-to-EL8_{context}"]
+ifdef::satellite[]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+endif::[]
+ifdef::katello,foreman-el[]
```

Comment:
@maximiliankolb Would it be better to use `ifndef:satellite[]` instead?

Reply:
@Lennonka I did a build preview with the `ifndef::satellite[]`and got this
![image](https://user-images.githubusercontent.com/91690569/172230172-be055f6d-e1e4-445a-acce-7b059d3d4e8a.png)
Looks like 2.5 is what would be meant for Satellite but after seeing that there is a ticket for **Upgrading Satellite from RHEL 7 to RHEL 8 with LEAPP** with accompanying documentation, can we make it so that none of this content including anything with `ifdef::{}`gets built into Satellite documentation? It just needs to be for Foreman/Katello.

Reply:
Yes, please apply Lena's suggestion. Something like:

```
if satellite:
  heading 1
if not satellite
  heading 2
```
Neither build targets should display empty chapters or redundant headings.